### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by sahayayush987** [minerllabs/minerl#590]
No idea what is going on. I'm completely new to this, and i am unable to figure out why things are not working

DEBUG:minerl.env.malmo.instance.d251ca:#################################################
DEBUG:minerl.env.malmo.instance.d251ca:         ForgeGradle 2.2-SNAPSHOT-8de58cf        
DEBUG:minerl.env.malmo.instance.d251ca:  https://github.com/MinecraftForge/ForgeGradle  
DEBUG:minerl.env.malmo.instance.d251ca:#################################################
DEBUG:minerl.env.malmo.instance.d251ca:               Powered by MCP unknown               
DEBUG:minerl.env.malmo.instance.d251ca:             http://modcoderpack.com             
DEBUG:minerl.env.malmo.instance.d251ca:         by: Searge, ProfMobius, Fesh0r,         
DEBUG:minerl.env.malmo.instance.d251ca:         R4wk, ZeuX, IngisKahn, bspkrs           
DEBUG:minerl.env.malmo.instance.d251ca:#################################################
DEBUG:minerl.env.malmo.instance.d251ca:Found AccessTransformer: malmomod_at.cfg
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :deobfCompileDummyTask
DEBUG:minerl.env.malmo.instance.d251ca:> Task :deobfProvidedDummyTask
DEBUG:minerl.env.malmo.instance.d251ca:> Task :getVersionJson
DEBUG:minerl.env.malmo.instance.d251ca:> Task :extractUserdev UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :downloadClient SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :downloadServer SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :splitServerJar SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :mergeJars SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :applyBinaryPatches SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :extractDependencyATs SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :extractMcpData SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :extractMcpMappings SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :genSrgs SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :deobfMcMCP
DEBUG:minerl.env.malmo.instance.d251ca:Applying SpecialSource...
DEBUG:minerl.env.malmo.instance.d251ca:Applying Exceptor...
DEBUG:minerl.env.malmo.instance.d251ca:Applying RuntimeInvisibleParameterAnnotations workaround...
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :sourceApiJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :compileApiJava NO-SOURCE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :processApiResources NO-SOURCE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :apiClasses UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :copyModToClient UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :copyModToServer UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :copySrg UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :deleteSchemas
DEBUG:minerl.env.malmo.instance.d251ca:> Task :copySchemas
DEBUG:minerl.env.malmo.instance.d251ca:> Task :jaxb UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :sourceMainJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :compileJava UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :processResources UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :classes UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :jar UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:> Task :extractNatives SKIPPED
DEBUG:minerl.env.malmo.instance.d251ca:> Task :getAssetIndex UP-TO-DATE
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :getAssets
DEBUG:minerl.env.malmo.instance.d251ca:Current status: 1029/1196   86%
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :makeStart
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :runClient
ERROR:minerl.env.malmo.instance.d251ca:Error occurred during initialization of VM
DEBUG:minerl.env.malmo.instance.d251ca:Could not reserve enough space for 2097152KB object heap
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Task :runClient FAILED
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:FAILURE: Build failed with an exception.
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:* What went wrong:
DEBUG:minerl.env.malmo.instance.d251ca:Execution failed for task ':runClient'.
DEBUG:minerl.env.malmo.instance.d251ca:> Process 'command 'C:\Program Files (x86)\Java\jdk1.8.0_301\bin\java.exe'' finished with non-zero exit value 1
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:* Try:
DEBUG:minerl.env.malmo.instance.d251ca:Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:* Get more help at https://help.gradle.org
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
DEBUG:minerl.env.malmo.instance.d251ca:Use '--warning-mode all' to show the individual deprecation warnings.
DEBUG:minerl.env.malmo.instance.d251ca:See https://docs.gradle.org/4.10.2/userguide/command_line_interface.html#sec:command_line_warnings
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:BUILD FAILED in 30s
DEBUG:minerl.env.malmo.instance.d251ca:20 actionable tasks: 9 executed, 11 up-to-date
DEBUG:minerl.env.malmo.instance.d251ca:
DEBUG:minerl.env.malmo.instance.d251ca:> Configure project :
DEBUG:minerl.env.malmo.instance.d251ca:This mapping 'snapshot_20161220' was designed for MC 1.11! Use at your own peril.

**Comment by Miffyli** [comment]
> Could not reserve enough space for 2097152KB object heap

This suggests you are running out of memory. Make sure you have installed a 64-bit version of the JDK and that you have at least 4GB of free memory left when building MineRL.

**Comment by sahayayush987** [comment]
Thank You! I downloaded x84 version of JDK. Now, i am trying to download the dataset, and it is asking me to do this. I am not sure that path is being asked here.

The minerl package uses the MINERL_DATA_ROOT environment variable to locate the data directory. Please export MINERL_DATA_ROOT=/your/local/path/.

**Comment by Miffyli** [comment]
Yes, as the instructions say, you should set the environment variable `MINERL_DATA_ROOT` to directory where you want to store the data.
